dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/RequestParamsValidator.java

```diff
@@ -313,7 +313,7 @@ public static void validateFilter(String filter, Set<Pair<String, Class<?>>> sup
     for (Pair<String, Class<?>> filterField : supportedFields) {
       if (filterField.getKey().equalsIgnoreCase(split[0])
           && filterField.getValue() == UID.class
-          && !CodeGenerator.isValidUid(filterField.getKey())) {
+          && !CodeGenerator.isValidUid(split[2])) {
```

Comment:
What test tests this fix?

Reply:
and now `RequestParamsValidatorTest.shouldFailWhenChangeLogFilterDoesNotHaveCorrectUidFormat` too
just that before it was validating the key instead of the value